### ISSUE TITLE
compatbility with php7

### ISSUE THREAD

**Issue by jsmota** [tstaerk/mediasyntax#23]
error message :
`Parse error: syntax error, unexpected 'new' (T_NEW) in *****\lib\plugins\mediasyntax\syntax\listblock.php on line 59`
workaround :
replace `& new` in line 59 and line 60 with `/*&*/ new`
reference : [PHP 7 and Wordpress: How to FIX unexpected &#039;new&#039; (T_NEW) error &#8902; CodeCave Blog](http://codecaveme.de/blog/en/php-7-and-wordpress-how-to-fix-unexpected-new-t_new-error/)
still some warnings

```
Warning: Declaration of syntax_plugin_mediasyntax_bold::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in *****\lib\plugins\mediasyntax\syntax\bold.php on line 47

Warning: Declaration of syntax_plugin_mediasyntax_bold::render($mode, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in *****\lib\plugins\mediasyntax\syntax\bold.php on line 47

Warning: Declaration of syntax_plugin_mediasyntax_codeblock::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in *****\lib\plugins\mediasyntax\syntax\codeblock.php on line 100

Warning: Declaration of syntax_plugin_mediasyntax_codeblock::render($mode, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in *****\lib\plugins\mediasyntax\syntax\codeblock.php on line 100

Warning: Declaration of syntax_plugin_mediasyntax_header::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in *****\lib\plugins\mediasyntax\syntax\header.php on line 72

Warning: Declaration of syntax_plugin_mediasyntax_header::render($mode, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in *****\lib\plugins\mediasyntax\syntax\header.php on line 72

Warning: Declaration of syntax_plugin_mediasyntax_include::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in *****\lib\plugins\mediasyntax\syntax\include.php on line 51

Warning: Declaration of syntax_plugin_mediasyntax_include::render($format, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in *****\lib\plugins\mediasyntax\syntax\include.php on line 51

Warning: Declaration of syntax_plugin_mediasyntax_italic::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in *****\lib\plugins\mediasyntax\syntax\italic.php on line 48

Warning: Declaration of syntax_plugin_mediasyntax_italic::render($mode, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in *****\lib\plugins\mediasyntax\syntax\italic.php on line 48

Warning: Declaration of syntax_plugin_mediasyntax_link::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in *****\lib\plugins\mediasyntax\syntax\link.php on line 56

Warning: Declaration of syntax_plugin_mediasyntax_link::render($mode, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in *****\lib\plugins\mediasyntax\syntax\link.php on line 56
```

**Comment by tstaerk** [comment]
thanks

**Comment by tstaerk** [comment]
tested fix, it works with PHP 5.5 as well && committed
